RazorGenerator.Core.v1/CodeTransformers/GenericParametersTransformer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.CodeDom;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace RazorGenerator.Core
+{
+    public class GenericParametersTransformer : RazorCodeTransformerBase
+    {
+        readonly string[] _parameters;
```

Comment:
Huh, I always assumed they had the same visibility as types viz `internal`. I'm going to try and figure out if there's a particular reason we have `private` decorators in all our code despite the guidelines @davidebbo's pointed to not having anything about these.